src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.NoSuchElementException;
+
+public class Calculator {
+
+    public double calculate(double x, double y, String operation) {
+        switch (operation) {
+            case "+":
+                return x + y;
+            case "-":
+                return x - y;
+            case "*":
+                return x * y;
+            case "/":
+                if (y == 0) {
+                    throw new ArithmeticException("Can't divide by zero...");
+                }
+                return x / y;
+            case "^":
+                return Math.pow(x, y);
+            default:
+                throw new NoSuchElementException("Your data is incorrect...");
```

Comment:
Давай краще кидатимемо наш власний виняток
```suggestion
                throw new InvalidOperationException("Invalid operation");
```